engine/src/main/java/org/terasology/rendering/ShaderManagerLwjgl.java

```diff
@@ -151,14 +145,13 @@ public void recompileAllShaders() {
         activeMaterial = null;
     }
 
-    @SuppressWarnings("unused")  // this is actually used, I'm not sure why the Jenkins Static Analyses says it's unused.
-    private GLSLMaterial prepareAndStoreShaderProgramInstance(String title, ShaderParameters params) {
+    // TODO: discuss having a `public unloadShaderProgram`, to dispose shader programs no longer in use by any node
+    public GLSLMaterial loadShaderProgram(String title) {
```

Comment:
I won't argue about this, you can leave it as it is, but add/remove are, in our project, a bit more standard than load/unload, especially when it's about adding something to a manager of some kind, like in this case. No biggie though.